### PR TITLE
refine test & tweak gitignore

### PR DESCRIPTION
**Pull request by zzz6519003**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -27,3 +27,4 @@ chrome-user-data
 .idea
 *.iml
 .vscode
+yarn.lock
```
Comment (yaycmyk) [inline .gitignore]: Why would you ignore yarn.lock? The whole point of it is to be included to ensure specific versions of things are installed.

Comment (aweary) [inline .gitignore]: Yes, please remove this, we explicitly made sure to check this in https://github.com/facebook/react/pull/7957

File: src/renderers/shared/fiber/__tests__/ReactIncremental-test.js [hunk 2]
```diff
@@ -58,11 +58,16 @@ describe('ReactIncremental', () => {
...
+    // Do first step of work.
     ReactNoop.flushDeferredPri(7 + 5);
     expect(fooCalled).toBe(true);
     expect(barCalled).toBe(false);
     expect(renderCallbackCalled).toBe(false);
+    // Do second step of work
+    ReactNoop.flushDeferredPri(7 + 5 + 5);
+    expect(fooCalled).toBe(true);
+    expect(barCalled).toBe(true);
+    expect(renderCallbackCalled).toBe(false);
```
Comment (aweary) [inline src/renderers/shared/fiber/__tests__/ReactIncremental-test.js]: @acdlite does this add any value here?

**Comment by aweary** [comment]
Since there's been no follow-up, and the test changes are non-critical (and possibly unwanted) I'm going to go ahead and close this out. Thanks for the effort though @zzz6519003!

**Pull request closed without merging**